snap/info.go

```diff
@@ -207,6 +207,8 @@ type Layout struct {
 	Group   string      `json:"group,omitempty"`
 	Mode    os.FileMode `json:"mode,omitempty"`
 	Symlink string      `json:"symlink,omitempty"`
+
+	BindFile string `json:"bind-file,omitempty"`
```

Comment:
nit: can you put it under `Bind`? or was there a reason (beyond whitespace-only diff) that you didn't?

(you can skip the whitespace-only bit in diffs with `-w` on the commandline, or with `?w=1` on github...)

Reply:
It was just for whitespace, I will move it up.